### PR TITLE
libaom: update to 3.2.0.

### PR DESCRIPTION
**Pull request by tibequadorian**
#### Testing the changes
- I tested the changes in this PR: **YES**

### CODE REVIEW

File: srcpkgs/libaom/template [hunk 1]
```diff
@@ -12,7 +13,9 @@ license="BSD-2-Clause"
 homepage="https://aomedia.org/"
 changelog="https://aomedia.googlesource.com/aom/+/master/CHANGELOG"
 distfiles="https://aomedia.googlesource.com/aom/+archive/v${version}.tar.gz"
-checksum=@b89acd3304531a9f6c99eba13e1de66605893ff8d08abc8c6e0586481a90033a
+checksum=@358ac353cb20555aa8ad8faa4d14b98bf89dd05f76c45548d7219673c767d247
+
+export CMAKE_GENERATOR="Unix Makefiles"
```
Comment (paper42) [inline srcpkgs/libaom/template:18]: why is this necessary?

Comment (tibequadorian) [reply srcpkgs/libaom/template:18]: It doesn't work with ninja. Tell me if there's another way to make it use make instead of ninja.

Comment (tibequadorian) [reply srcpkgs/libaom/template:18]: More precisely, the tests don't work with ninja when I remember correctly.

Comment (tibequadorian) [reply srcpkgs/libaom/template:18]: Now I know: Until now the `do_check` function calls `make` for testing. But because we use `ninja` instead of `make` this fails, so I changed `make` to `${make_cmd}`. This fails too, because the tests doesn't work with `ninja`, so I set `make_cmd=make` and `export CMAKE_GENERATOR="Unix Makefiles"` :)

Comment (paper42) [reply srcpkgs/libaom/template:18]: `ninja -C build testdata` seems to work for me.

Comment (tibequadorian) [reply srcpkgs/libaom/template]: sorry i am a little lost at this :/ don't know why it doesn't work
